tests/Microsoft.DotNet.Framework.Docker.Tests/projects/aspnet-4.8/Dockerfile

```diff
@@ -2,6 +2,8 @@ ARG BASE_ASPNET_IMAGE
 
 FROM $BASE_ASPNET_IMAGE
 
+RUN Add-WindowsFeature Web-AppInit -IncludeAllSubFeature
```

Comment:
This does the trick.  Please apply this change to the rest of the ASP.NET and WCF test project Dockerfiles except for 3.5 where this command is returning an error code.  That's ok, though, because I don't see timeout issues for 3.5.